docroot/sites/all/themes/ilr_theme/js/ilr_theme.js

```diff
@@ -404,4 +404,22 @@
     }
   };
 
```

Comment:
Probably. I've been doing a lot of js recently that I probably overlooked the obvious.